### PR TITLE
feat(config): Expose DisableCache as an environment variable

### PR DESCRIPTION
**Pull request by Pokom**
This adds the ability to set the value for `disableCache` via an environment variable `DISABLE_CACHE`. The default value is set to false. This will influence whether ComputeAggregateCostModel will use data if it's found in the cache.

- Relates to #1472

Signed-off-by: pokom <mark.poko@grafana.com>

## What does this PR change?
* Adds an environment variable `DISABLE_CACHE` that defaults to false

## Does this PR relate to any other PRs?
* #1473

## How will this PR impact users?
* Users can now *opt in* to disabling the cache

## Does this PR address any GitHub or Zendesk issues?
* 

## How was this PR tested?
* Added unit tests to ensure the value defaults to false and can be updated

## Does this PR require changes to documentation?
* Most likely need to add `DISABLE_CACHE` to environment variables that can be overriden

## Have you labeled this PR and its corresponding Issue as "next release" if it should be part of the next Opencost release? If not, why not?
*

### CODE REVIEW

**Comment by AjayTripathy** [comment]
@nikovacevic for review,  think this makes sense to bring back as an option in light of increased usage of the direct open source APIs?

**Comment by nikovacevic** [comment]
Yeah, I'm not opposed. But we'll need a more specific name for the cache.

File: pkg/env/costmodelenv.go [hunk 1]
```diff
@@ -239,6 +240,12 @@ func GetInsecureSkipVerify() bool {
 	return GetBool(InsecureSkipVerify, false)
 }
 
+// IsCacheDisabled returns the environment variable value for DisableCache which
+// will inform the aggregator on whether to load cached data. Defaults to false
+func IsCacheDisabled() bool {
```
Comment (nikovacevic) [inline pkg/env/costmodelenv.go]: @Pokom would you be open to renaming this function and the associated variable `IsAggregateCostModelCacheDisabled()` and `DISABLE_AGGREGATE_COST_MODEL_CACHE`, respectively?

Comment (Pokom) [reply pkg/env/costmodelenv.go]: Certainly! I just pushed a commit to update the references to be explicit in what cache is being set. In addition to the method definitions, I updated the other `disableCache` variables to be `disableAggregateCostModelCache`. Let me know if that's a problem!

Comment (nikovacevic) [reply pkg/env/costmodelenv.go]: Not at all -- very helpful, thank you.

**Review by nikovacevic — APPROVED** [review]

**Comment by nikovacevic** [comment]
@AjayTripathy feel free to take a final pass if you like, or merge when ready

**Pull request merged**